### DISCUSSION TITLE
Can you add "skip intro" button pls? if no, why?

### DISCUSSION THREAD

**Discussion by Osu-Player** [ppy/osu#29045] [Ideas] [answered]
https://github.com/ppy/osu/discussions/23933
It annoys me a little that, seeing that the Osu(not lazer) is fully loaded, I have to watch these ~3-4 seconds of intro. I know many people with this problem and, if it possible, we want skip button for it or the item in the settings "skip the entry when entering the game"
(the item?.. google translate...)

**Comment by Joehuu** [comment, marked as answer]
This was asked before in https://github.com/ppy/osu/discussions/15261.

**Reply by Osu-Player** [reply]
thx

**Comment by peppy** [comment]
As a temporary workaround, you can set this when creating the multiplayer lobby itself.

![osu! 2024-07-25 at 03 40 53](https://github.com/user-attachments/assets/02663c9c-bf96-4092-9774-21a7cad81b44)

**Reply by Osu-Player** [reply]
thx, but i can't play in osu!lazer(my computer is weak, and the lazer requires a good processor)
